src/ProfilerEvent.php

```diff
@@ -58,7 +58,7 @@ public function setApplication(ApplicationInterface $application)
      *
      * @return string
      */
-    public function getProfiler()
+    public function getProfiler(): string
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!